### DISCUSSION TITLE
HTML example is not working

### DISCUSSION THREAD

**Discussion by janiomaria** [WalletConnect/walletconnect-monorepo#3445] [Developer Support] [answered]
I've tried these 2 examples:
https://github.com/WalletConnect/web3modal-examples/tree/main/web3modal-wagmi-html-cdn
https://github.com/WalletConnect/web3modal-examples/tree/main/walletconnect-modal-sign-html-cdn
Both of them do not work, I click button and nothing happens for walletconnect and there's no any button for web3modal-wagmi
Tried with
`npx http-server`
and
`python -m http.server`
Same examples did work for me like 10 days ago, idk what happened now. Maybe it has something to do with new "Domain Verify" on walletconnect, please help

**Comment by glitch-txs** [comment, marked as answer]
answered [here](https://github.com/orgs/WalletConnect/discussions/3450) :)
